leku/src/main/java/com/schibstedspain/leku/locale/DefaultCountryLocaleRect.kt

```diff
@@ -51,6 +51,10 @@ internal object DefaultCountryLocaleRect {
     private val BRAZIL_UPPER_RIGHT = LatLng(10.236344, -30.084353) // Brazil
     private const val BRAZIL_COUNTRY_CODE = "pt_BR"
 
+    private val IRELAND_LOWER_LEFT = LatLng(51.527021, -10.237684)
```

Comment:
maybe this coordinates are too narrow and could lead to not to fully cover all the territory. I suggest you to use a coordinates more far from the land

Reply:
Yeah sure. I'll update my PR now. Thanks!